dns-policy-operator/app/policy/management/commands/create_reviewer.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Create reviewer command."""
+
+import select
+import getpass
+import sys
+
+from django.contrib.auth.models import Group, User
+from django.contrib.auth.password_validation import validate_password
+from django.core import exceptions
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    """Create reviewer command."""
+
+    help = 'Create a new reviewer'
+
+    def add_arguments(self, parser):
+        """Define CLI arguments."""
+        parser.add_argument('username', type=str, help='Username of the user')
+        parser.add_argument('email', type=str, help='Email of the user')
+
+    def handle(self, *args, **options):
+        """Handle CLI invocation."""
+        # Check if there is any input available on sys.stdin
+        rlist, _, _ = select.select([sys.stdin], [], [], 0)
+        if rlist:
+            # Read password from STDIN
+            password = sys.stdin.readline().strip()
+        else:
+            # Took example on Django here
+            # ref: https://github.com/django/django/blob/main/django/contrib/auth/management/commands/createsuperuser.py#L172
+            password = getpass.getpass()
+            password_repeat = getpass.getpass("Password (again): ")
+            if password != password_repeat:
+                self.stderr.write("Error: Your passwords didn't match.")
+                # Don't validate passwords that don't match.
+                return
+
+        if password.strip() == "":
+            self.stderr.write("Error: Blank passwords aren't allowed.")
+            # Don't validate blank passwords.
+            return
+        try:
+            validate_password(password)
+        except exceptions.ValidationError as err:
+            self.stderr.write("\n".join(err.messages))
+            return
+
+        user, created = User.objects.get_or_create(
+            username=options['username'],
+            defaults={
+                'email': options['email'],
```

Comment:
Out of curiosity: Is email field unique?

Reply:
No [it's not](https://github.com/django/django/blob/efec74b90868c2e611f863bf4301d92ce08067e8/django/contrib/auth/models.py#L470). I'm using Django's User